projects/pgai/pgai/vectorizer/chunking.py

```diff
@@ -52,19 +58,14 @@ class LangChainCharacterTextSplitter(BaseModel, Chunker):
     """
 
     implementation: Literal["character_text_splitter"]
-    separator: str
-    chunk_size: int
-    chunk_column: str
-    chunk_overlap: int
-    is_separator_regex: bool
 
     @cached_property
     def _chunker(self) -> CharacterTextSplitter:
         return CharacterTextSplitter(
-            separator=self.separator,
+            separator=self.separator,  # type: ignore
```

Comment:
This `type: ignore` is now necessary as pyright does not know that the decorator makes the field required and complains about possibly passing `None`.